app/controllers/account_activations_controller.rb

```diff
@@ -10,13 +10,19 @@ def show
   
   def edit
     user = User.find_by(email: params[:email].downcase)
-    if user && !user.activated? && @user.authenticated?(:activation, params[:id])
+    if user && !user.activated? && @user.authenticated?(:activation, activation_token)
       user.activate
       
       log_in user
-      
-      flash[:success] = t('.account_activated')
-      redirect_to root_url
+      if !!cookies.encrypted[:game_invite_code]
```

Comment:
`!!cookies.encrypted[:game_invite_code]` is functionally the same as `cookies.encrypted[:game_invite_code]`. If you want to make this a bit more understandable, I would use `if cookies.encrypted[:game_invite_code].present?`.